napari/_vispy/layers/points.py

```diff
@@ -131,15 +130,13 @@ def _on_highlight_change(self):
             or 0 in self.layer._highlight_box.shape
         ):
             pos = np.zeros((1, self.layer._slice_input.ndisplay))
-            width = 0
```

Comment:
This change is responsible for the example crashing reported:
https://napari.zulipchat.com/#narrow/stream/212875-general/topic/vispy.20.22Last.20dimension.20should.20be.202.20not.203.22.20error

@dalthviz you can try `highlight_thickness = 0` here maybe this also fixes https://github.com/napari/napari/pull/6819#discussion_r1562964127

Reply:
Ahn... For some reason I though it was initialized at 0 and this was a noop... My bad :/

Reply:
Checked and seems like for https://github.com/napari/napari/pull/6819#discussion_r1562964127 the change is not enough but thank you for letting me know!